### ISSUE TITLE
Graph Capsule Networks

### ISSUE THREAD

**Issue by jlevy44** [rusty1s/pytorch_geometric#233]
Are there planned implementations of these?

**Comment by rusty1s** [comment]
Would love a pull request on this one.

**Comment by jlevy44** [comment]
Yeah would be great to see a PR. Unfortunately, I'm probably too busy to give the PR a shot, but if no one gets to it, I'll give it a shot.
